bower.json

```diff
@@ -27,7 +27,8 @@
     "jed": "0.5.0",
     "escope": "~1.0.0",
     "iframe-overlay": "0.1.4",
-    "poster": "0.1.0"
+    "poster": "0.1.0",
+    "dialog-polyfill": "~0.2.0"
```

Comment:
only Chrome supports the <dialog> tag so let's polyfill it (it's smaller than jquery-ui's modal)